ci/jobs/satellite6-python-trigger.yaml

```diff
@@ -0,0 +1,88 @@
+- job:
+    name: satellite6-pulp-trigger
+    node: f23-vanilla-np
+    wrappers:
+        - ansicolor
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-2.7
+            system-site-packages: true
```

Comment:
It is probably to install requests. You can do the following:

Add a build step before like this:

```
- shining-pand:
    name: satellite6-pulp-trigger
    python-version: System-CPython-2.7
    nature: shell
    command: |
        pip install -U pip
        pip install requests
```

Then remove the `system-site-packages: true` and add a name property to this build step with the same name as the previous one (`name: satellite6-pulp-trigger`).